database/models.py

```diff
@@ -0,0 +1,36 @@
+### models.py ###
+
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import Column, Integer, String
+from sqlalchemy.sql.schema import ForeignKey
+
+Base = declarative_base()
+
+
+class Release(Base):
+    __tablename__ = "releases"
+
+    id = Column(Integer, primary_key=True)  # Unique arbitrary int assigned when input
+    version = Column(String)  # expected to be in form 20XX.YY
+    result = Column(String)  # expected to be "success", "failed", or "in progress"

```

Comment:
I love the code comments 😃 this is really helpful for knowledge sharing.
Great job 👍🏼